libcloudforensics/providers/azure/internal/common.py

```diff
@@ -80,7 +163,7 @@ def ExecuteRequest(
     request = getattr(client, func)
     response = request(**kwargs)
     responses.append(response)
-    next_link = response.next_link
+    next_link = response.next_link if hasattr(response, 'next_link') else None
```

Comment:
See `getattr`: https://docs.python.org/3/library/functions.html#getattr and its default return value
```suggestion
    next_link = response.getattr('next_link', None)
```